benchmarking/BENCHMARKS.log

```diff
@@ -0,0 +1,2 @@
+Date/Time                 	uvsim_version	SettingsFile 	cpus-per-task	Ntasks	Nnodes	MemLimit	Ntimes	Nbls	Nfreqs	Nsrcs	Nsrcs_part	MaxRSS [GiB]	Runtime
+2020-03-06-21:40:47.584612	1.1.2.       	settings.yaml	2            	10    	1     	50G     	21    	500 	256   	3072 	1         	13.071      	4:32:16.373965
```

Comment:
Is this all the info we need to tell if there are speed changes? I guess I'm worried about whether different lines could be wildly different if something about the hardware is different. Maybe just adding the hostname would help with tracking that?

Reply:
I've added the host machine name and url, which should be enough to indicate where the job was run. It's just one more column. I'd like to keep this file readable, so I'm hesitant to add too many more columns.

Reply:
that sounds good, thanks!